facade/nodeFacade.go

```diff
@@ -552,6 +552,16 @@ func (nf *nodeFacade) convertVmOutputToApiResponse(input *vmcommon.VMOutput) *vm
 	}
 }
 
+// GetGenesisNodesPubKeys will return genesis nodes public keys by shard
+func (nf *nodeFacade) GetGenesisNodesPubKeys() (map[uint32][][]byte, map[uint32][][]byte, error) {
+	eligible, waiting := nf.apiResolver.GetGenesisNodesPubKeys()
+	if eligible == nil || waiting == nil {
```

Comment:
might change the condition to `eligible == nil && waiting == nil` as if we'd have for example only eligible and no waiting, then you'd return error but it would be better to see the actual eligible and a `null` for waiting

Reply:
modified the condition :+1: